### DISCUSSION TITLE
PaddleOCR/algorithm/table_recognition/algorithm_table_slanet

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#13672] [Q&A] [answered]
# PaddleOCR/algorithm/table_recognition/algorithm_table_slanet

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/algorithm/table_recognition/algorithm_table_slanet.html

**Comment by myhloli** [comment]
推理模型下载地址不见了？我记得之前这个有可以直接用的推理模型来着。

**Reply by GreatV** [reply, marked as answer]
在pr里面，还没合入。

**Comment by lanhong123** [comment]
为什么我使用的效果这么差

**Reply by lanhong123** [reply]
会出现一大堆格子，是我权重用错了吗‘’

**Comment by yoliax** [comment]
如何转onnx呢？

paddle2onnx --model_dir /workspace/models/ch_ppstructure_openatom_SLANetv2_infer \
            --model_filename inference.pdmodel \
            --params_filename inference.pdiparams \
            --enable_onnx_checker True \
            --opset_version 12 \
            --save_file /workspace/models/ch_ppstructure_openatom_SLANetv2_infer/model.onnx

报错
[Paddle2ONNX] Start to parse PaddlePaddle model...
[Paddle2ONNX] Model file path: /workspace/models/ch_ppstructure_openatom_SLANetv2_infer/inference.pdmodel
[Paddle2ONNX] Parameters file path: /workspace/models/ch_ppstructure_openatom_SLANetv2_infer/inference.pdiparams
[Paddle2ONNX] Start to parsing Paddle model...
[Paddle2ONNX] Oops, there are some operators not supported yet, including conditional_block,select_input,
[ERROR] Due to the unsupported operators, the conversion is aborted.

按照paddle2onnx 指定1.2.5版本也转换不成功
